client/src/components/Home.js

```diff
@@ -1,127 +1,83 @@
-                      >
-                        <h1>{element.person}</h1>
-                        <h3>{element.title}</h3>
-                        <p>{element.testimonial}</p>
-                      </Col>
-                    </Row>
-                  </div>
-                );
-              })}
-            </Carousel>
-          </Col>
-        </Row>
-        <Row
-          className="home-block-4"
-          type="flex"
-          justify="center"
-          align="middle"
-        >
-          <Col>
-            <Row type="flex" justify="center" align="middle">
-              <Col>
-                <h1>Partners in the Community</h1>
-              </Col>
-            </Row>
-            <Row type="flex" justify="center" align="middle">
-              <Col className="home-block-4__partner" span={4}>
-                <img
-                  src="/asset/partners/cu_fair.jpg"
-                  alt="Champaign-Urbana Fair"
-                />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img
-                  src="/asset/partners/cuphd.png"
-                  alt="Champaign-Urbana Public Health District"
-                />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img src="/asset/partners/dhai_tree.jpg" alt="Dhairtree" />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img
-                  src="/asset/partners/three_spinners.jpg"
-                  alt="Three Spinners"
-                />
-              </Col>
-              <Col className="home-block-4__partner" span={4}>
-                <img src="/asset/partners/trc.jpg" alt="The Refugee Center" />
-              </Col>
-            </Row>
-          </Col>
-        </Row>
-      </>
-    );
-  }
-}
+import { HomeBlock2Desktop, HomeBlock3Desktop } from './desktop/HomeDesktop';
+import { HomeBlock2Mobile, HomeBlock3Mobile } from './mobile/HomeMobile';
+
+const Home = () => {
+  const [{ height, width }, isMobile] = useWindowDimensions();
+
+  return (
+    <>
+      <Row className="home-block-1" type="flex" justify="center" align="middle">
+        <Col span={6}>
+          <Textfit className="welcome-text" mode="single">
+            Welcome to
+          </Textfit>
+          <Textfit className="welcome-text" mode="single">
+            Urbana-Champaign
+          </Textfit>
+          <Row type="flex" justify="center" align="middle">
+            <Col span={18}>
+              <Link to="/resources">
+                <Button type="primary">
+                  <strong>Find Resources</strong>
+                </Button>
+              </Link>
+            </Col>
+          </Row>
+        </Col>
+      </Row>
+      {!isMobile && <HomeBlock2Desktop />}
+      {isMobile && <HomeBlock2Mobile />}
+      {!isMobile && <HomeBlock3Desktop />}
+      {isMobile && <HomeBlock3Mobile />}
```

Comment:
We can use [conditional rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator) here. So the code might look like:

```js
{isMobile ? (
    <>
        <HomeBlock2Mobile>
        <HomeBlock3Mobile>
    </>
    ) : (
    <>
        <HomeBlock2Desktop>
        <HomeBlock3Desktop>
    </>
    )
}
```

This will make it easier to determine mobile or not, and add/remove components if necessary in the future

Reply:
sounds good!